web/src/tippyjs.js

```diff
@@ -304,6 +304,18 @@ export function initialize() {
         },
     });
 
+    delegate("body", {
+        target: ["#api_key_button_container.disabled_setting_tooltip"],
+        content: $t({
+            defaultMessage:
+                "Demo organization owners must configure their email to access this feature.",
```

Comment:
This message seems not directly correctly; the person seeing this will be doing a demo, and probably something like "The Zulip API is only available to accounts with an email address configured."

Reply:
Or maybe something more like "You must configure your email address to access this feature" if we want it to be generic; my main concern here is about "Demo organization owners" being a bit too jargony.